cli/smartcontract/smart_contract.go

```diff
@@ -89,8 +89,8 @@ func NewCommands() []cli.Command {
 						Usage: "Output of the compiled contract",
 					},
 					cli.BoolFlag{
-						Name:  "debug, d",
-						Usage: "Debug mode will print out additional information after a compiling",
+						Name:  "verbose, v",
+						Usage: "Print out additional information after a compiling",
 					},
 					cli.StringFlag{
 						Name:  "emitdebug",
```

Comment:
BTW, this whole change was intended to free `debug, d` for this option, so why isn't it used here?

Reply:
ping

Reply:
Done.